z.psm1

```diff
@@ -102,7 +111,7 @@ function z {
 
     # If a valid path is passed in to z, treat it like the normal cd command
     if (-not $ListFiles -and -not [string]::IsNullOrWhiteSpace($JumpPath) -and (Test-Path $JumpPath)) {
-        if ($Push) {
+        if ($Push -or $Z_UsePushLocation) {
```

Comment:
Could just use `$Z_UsePushLocation` as the default value for the `$Push` parameter, instead of hard-coded `$false`.

Reply:
Sorry, was referring to this bit in this method:

```
    [Alias('p')]
    [switch]
    $Push = $false
```

You could do this instead:

```
    [Alias('p')]
    [switch]
    $Push = $Z_UsePushLocation
```

That way you can avoid checking it with `-or $Z_UsePushLocation` in the method body.